src/main/java/karen/commands/Command.java

```diff
@@ -0,0 +1,20 @@
+package karen.commands;
+
+import karen.tasks.TaskList;
+import karen.util.Ui;
+
+public abstract class Command {
+
+    /**
+     * Executes this {@code Command}
+     * @param taskList TaskList to execute command on
+     * @param ui Ui object to handle user input/output
+     */
+    public abstract void execute(TaskList taskList, Ui ui);
+
+    /**
+     * Checks to see if this {@code Command} is of type {@code ExitCommand}
+     * @return boolean
+     */
+    public abstract boolean isExit();
```

Comment:
Maybe using a verb would be better as a name for this method? 🤔 
`checkExit` might work, since `isExit` feels more like a boolean variable.